database/migrations/model/Version20221012133111.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Migrations\Model;
+
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+
+class Version20221012133111 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $sql = <<<SQL
+ALTER TABLE SummitAttendee MODIFY AdminNotes VARCHAR(1024) CHARACTER SET utf8 COLLATE utf8_unicode_ci NULL;
```

Comment:
this is not correct
check the collation of the DB
utf8mb4 ( there are Chinese kanjis that needs 4 bytes)
regards

Reply:
also please check that migration is compatible with mysql 5.7

Reply:
fixed